vmm_tests/vmm_tests/tests/tests/aarch64.rs

```diff
@@ -15,3 +15,12 @@ async fn frontpage(config: PetriVmConfig) -> anyhow::Result<()> {
     assert_eq!(vm.wait_for_teardown().await?, HaltReason::PowerOff);
     Ok(())
 }
+
+/// Basic boot test.
+#[vmm_test(uefi_aarch64(vhd(ubuntu_2404_server_aarch64)))]
```

Comment:
Can we start a multiarch module, and move the x86 and this boot test into it?

Reply:
Yeah sounds good